Utilities/Pipelines/libHttpClient.CI.yml

```diff
@@ -0,0 +1,168 @@
+# CI trigger. Fires on merges to specific branches.
+# Ref: https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/azure-repos-git#ci-triggers
+# A pipeline with no CI trigger
+trigger: none
+
+# Nightly trigger. Builds on the specified schedule.
+# Refs:
+# - https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers
+# - https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers#cron-syntax
+schedules:
+  # Schedule is specified in `cron` syntax. The below means: "8am UTC (midnight PST) every day".
+  - cron: "0 8 * * *"
```

Comment:
Since this is a nightly build but has no trigger, let's name the file (and pipeline based on this def) to `libHttpClient.Nightly.yml` maybe? Alternatively, could be good to set a trigger for the `main` branch, so this runs on merges.

Reply:
Should have a trigger.  I'll add